src/app/shared/dialog-ministry-plates/dialog-ministry-plates.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit, ChangeDetectionStrategy, Inject } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { DialogBoxComponent } from '@app/shared/dialog-box/dialog-box.component';
+import { Applicant, Plate } from '@app/models/tech-record.model';
+
+export interface DialogData {
+  extra: Applicant;
+}
+
+@Component({
+  selector: 'vtm-dialog-ministry-plates',
+  templateUrl: './dialog-ministry-plates.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
```

Comment:
`exportAs: plateComponent`

Reply:
Closing as I added exportAs in ministry-plates.component.ts
![image](https://user-images.githubusercontent.com/55185857/86338928-9e2cbf80-bc5b-11ea-9625-b0c312bc57ee.png)